MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift

```diff
@@ -24,7 +27,17 @@ class DetailsViewController: UIViewController {
         abilitiesView.addSubview(childView.view)
         abilitiesView.addConstrained(subview: childView.view)
         childView.didMove(toParent: self)
-    }
+        
+        
+        let movieCustomView = MovieSectionView(character: character!, frame: CGRect(x: 0, y: 0, width: 0, height: 0))
```

Comment:
We should avoid force unwrapping with "!", use guard let instead

Reply:
Fixed